msgpack/_packer.pyx

```diff
@@ -241,10 +242,10 @@ cdef class Packer(object):
                 if PyObject_GetBuffer(o, &view, PyBUF_SIMPLE) != 0:
                     raise PackValueError("could not get buffer for memoryview")
                 L = view.len
-                if L > (2**32)-1:
+                if L > ITEM_LIMIT:
                     PyBuffer_Release(&view);
                     raise PackValueError("memoryview is too large")
-                ret = msgpack_pack_bin(&self.pk, L)
+                ret = msgpack_pack_raw(&self.pk, L)
```

Comment:
I misread the code of ``msgpack_pack_raw``, this shouldn't have changed. Reverted it now. Thanks!